src/test/java/uk/ac/ebi/ampt2d/accession/sha1/SampleSha1AccessioningServiceTest.java

```diff
@@ -0,0 +1,160 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.accession.sha1;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.ac.ebi.ampt2d.accession.AccessioningRepository;
+import uk.ac.ebi.ampt2d.accession.AccessioningService;
+import uk.ac.ebi.ampt2d.accession.WebConfiguration;
+import uk.ac.ebi.ampt2d.accession.sample.SampleMessage;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+@TestPropertySource(properties = "services=sample-accession")
+@ComponentScan(basePackageClasses = AccessioningRepository.class)
+@ContextConfiguration(classes = WebConfiguration.class)
```

Comment:
Please
 +@ComponentScan(basePackageClasses = AccessioningRepository.class)
+@ContextConfiguration(classes = WebConfiguration.class)

Change this to the approach used in my PR to avoid a rage explosion from my part with Travis.

Reply:
ya i  have changed it , waiting for monotonic be merged. ill rebase and checkin the code.